data/brands/office/company.json

```diff
@@ -4,6 +4,17 @@
     "exclude": {"generic": ["^company$"]}
   },
   "items": [
+    {
+     "displayName": "Salesforce",
+     "locationSet": {"include": ["001"]},
+     "tags": {
+       "brand": "Salesforce",
+       "brand:wikidata": "Q941127",
```

Comment:
Salesforce isn’t a brand of IT company offices. They operate IT offices, so the `operator` key would be more appropriate than the `brand` key.